lib/mongo_client.js

```diff
@@ -191,16 +192,38 @@ const validOptions = require('./operations/connect').validOptions;
  * @param {MongoClientOptions} [options] Optional settings
  */
 function MongoClient(url, options) {
+  options = options || {};
   if (!(this instanceof MongoClient)) return new MongoClient(url, options);
   // Set up event emitter
   EventEmitter.call(this);
 
   if (options && options.autoEncryption) require('./encrypter'); // Does CSFLE lib check
```

Comment:
```suggestion
  if (options.autoEncryption) require('./encrypter'); // Does CSFLE lib check
```
Now that we're defaulting options, just a little cleanup.